coordinator/src/services/translation/spine-response.ts

```diff
@@ -86,7 +86,7 @@ function getFhirResponseAndErrorCodes<T extends AsyncMCCI | SyncMCCI>(
 
 function createOperationOutcomeIssue(
   statusCode: number,
-  hl7Message: string,
+  hl7Message?: string,
```

Comment:
Is there a reason we're still passing this in on line 76 & 77? I'd have thought we'd want to get rid of it in all places

Reply:
i was only getting rid of it on successful prescription-order, but on second thoughts i'll remove it entirely